native_client/javascript/index.js

```diff
@@ -35,7 +35,8 @@ function Model(aModelPath) {
     const status = rets[0];
     const impl = rets[1];
     if (status !== 0) {
-        throw "CreateModel failed with error code 0x" + status.toString(16);
+        error_message = binding.ErrorCodeToErrorMessage(status);
+        throw "CreateModel failed with error message "+error_message+" with error code 0x" + status.toString(16);
```

Comment:
Same nit

Reply:
@imskr I'm thinking it could a good think to verify all call sites of the API in bindings and make sure they all properly at least show a meaningful error message. Bonus point for making that language-consistent (throwing exception, etc.)

Reply:
Yes that would be right